subsys/net/lib/sockets/getaddrinfo.c

```diff
@@ -51,6 +51,7 @@ struct getaddrinfo_state {
 	int status;
```

Comment:
Please add `net: ` prefix to commit subject. Also I don't think it is needed to put back-ticks \` to commit message.

Reply:
Thanks, I've updated the commit message.

Reply:
Still a typo in "exectued"